### ISSUE TITLE
Sigma Dimensions and Floating Buildings

### ISSUE THREAD

**Issue by Joker58th** [Sigma88/Sigma-Dimensions#65]
## Disclaimer by Sigma88
This is not an issue with SD itself, but rather an issue with the idea of rescaling in general. SD offers different ways to solve the issues of rescaling planets and PQSCity mods, but an automatic solution is impossible. In this thread you will find some suggestions on how to approach these issues using proper SD syntax

<br><b>Original message by @Joker58th :</b>

I looked over some forum posts and eberkain's issue which seemed similar but I am unable to get a config that fixes the floating issue with the 3.2 or 6.4 rescale.

https://i.imgur.com/49RDsGa.png

I have a fresh 1.3.1 install with the following:

Mods
https://i.imgur.com/5dgUAZI.png

Gamedata
https://i.imgur.com/WQvukTg.png

Config
https://i.imgur.com/DsTYIcw.png

Any help would be appreciated.  Thanks.

**Comment by Sigma88** [comment]
there are a lo of problems with rescaling planets, I tried to account for all possibilities but there will always be the need of a certain management seen as how statics are positioned

the major problem here is that KK groups contain statics that are too scattered around and so SD pick one at random as the reference point, but the one chosen might not be the best option

I don't have a cfg that will fix it all for you, what I can do is help you on a "group-by-group" basis

basically: you take into consideration one group at a time and send me some data, and I will take a look at how to fix it

a trick that has worked in the past for a certain number of groups is to change the group of the ground station (sometimes there is a groundstation pretty far from a group but still included in that group, changing the group of that single static seems to solve some of the problems)


this could get very technical, so.. unless you are ready to do a lot of debugging, it might not be worth the effort


an easy alternative would be to set the "resizeBuildings" to whatever the resize of your planet is
this will basically make all the buildings bigger, but they should be in the right position

**Comment by Joker58th** [comment]
Thanks for the response Sigma.  I saw that Eskandare released Kerbinside Remastered and it seems to be working much better.  I only had to apply the "Center" config for a 3.2 rescale.  Its not perfect but its real close.  Thanks for the offer to help.  Much appreciated.

Just for my understanding, where do you set the "resizeBuildings" at?

**Comment by Sigma88** [comment]
it is one of the SD Settings

you can find them in GameData/Sigma/Dimensions/Settings.cfg

but the best way would be to edit that using a patch

like:

```
@SigmaDimensions
{
    %resizeBuildings = 3.2
}```

and put that in a cfg anywhere in gamedata

**Comment by Joker58th** [comment]
Cool, thanks for that info.  Also, just an FYI, but I saw that eberkain released his configs for 3.2x so I tried those with my setup and had the same floating issue.  Thankfully eberkain listed the mods and versions he used in the configs at the time he created them.  Through trying version by version, I discovered that any version of kerbal konstructs after 1.2.9.6 causes the floating issue.   I'm not sure whose problem that would be but I thought I'd let you know in case it ever comes up.  Or maybe you and GER-Space could touch base about it.  Thanks again for all you do.
